README.md

```diff
@@ -126,26 +126,26 @@ cd scripts
 python generate.py \
 --model open_lm_1b \
 --checkpoint /path/to/openlm_checkpoint.pt \
---rotary-old \
+--positional_embedding_type head_rotary \
 --input-text "Please give me a recipe for chocolate chip cookies"
 ```
-Again, note that `--rotary-old` is only necessary for the pretrained `open_lm_1b` model hosted below. 
+Again, note that `--positional_embedding_type head_rotary` is only necessary for the pretrained `open_lm_1b` model hosted below. 
 
 # Pretrained Models
 
 ## [OpenLM 1B](https://huggingface.co/mlfoundations/open_lm_1B)
 OpenLM 1B is a ~1Billion parameter model trained on a 1.6T token dataset which consists of a mix of RedPajama, Pile, S2ORC, The Pile of Law, Deepmind Math, and RealNews (the full mixture of training data is described in [more detail here](https://docs.google.com/spreadsheets/d/1YW-_1vGsSPmVtEt2oeeJOecH6dYX2SuEuhOwZyGwy4k/edit?usp=sharing)).
 The model checkpoint can be downloaded from [HuggingFace here](https://huggingface.co/mlfoundations/open_lm_1B/tree/main).
 The script used to train this model (for config-copying purposes) is [located here](https://github.com/mlfoundations/open_lm/blob/main/scripts/train_example.sh).
-Once this checkpoint has been downloaded, you can evaluate it by following the directions in the [Evaluate Model](#evaluate-model) section above and passing `--rotary-old` (see note below).
+Once this checkpoint has been downloaded, you can evaluate it by following the directions in the [Evaluate Model](#evaluate-model) section above and passing `--positional_embedding_type head_rotary` (see note below).
 
 Note: We trained this model with rotary embeddings applied to the _head_
 dimension, which is the default in xformers as of 09/01/2023. Since these models
 were trained, we have updated openlm to correctly apply the rotary embeddings to
 the sequence dimension (see
 [this issue](https://github.com/mlfoundations/open_lm/issues/4) and [this
 issue](https://github.com/facebookresearch/xformers/issues/841) for details).
-To evaluate these models, ensure you pass `--rotary-old` to the eval command.
+To evaluate these models, ensure you use the `m1b_neox_head_rotary.json` config.
```

Comment:
Should also be `--positional_embedding_type head_rotary`

Reply:
I should add that in the json file but m1b_neox_head_rotary does not exist... I can create it.

Reply:
We deleted the head_rotary configs now, and the rotary flag is always just done with the command line flags, so no need to add